ios/RCTWeChat.m

```diff
@@ -238,13 +236,12 @@ - (void)shareToWeixinWithData:(NSDictionary *)aData
 
         } else if ([type isEqualToString:RCTWXShareTypeImageFile] || [type isEqualToString:RCTWXShareTypeImageResource]) {
             NSString * imageURL = aData[RCTWXShareImageUrl];
-            
             [self.bridge.imageLoader loadImageWithTag:imageURL callback:^(NSError *error, UIImage *image) {
                 if (image == nil){
                     callback(@[@"fail to load image resource"]);
                 } else {
                     WXImageObject *imageObject = [WXImageObject object];
-                    imageObject.imageData = RCTGetImageData([image CGImage], 1.0F);
+                    imageObject.imageData = UIImageJPEGRepresentation(image, 1.0F);
```

Comment:
Hi @yorkie 
I've found the RCTGetImageData is extern at https://github.com/facebook/react-native/blob/master/Libraries/Image/RCTImageUtils.h#L79

and is defined at https://github.com/facebook/react-native/blob/master/Libraries/Image/RCTImageUtils.m#L285-L306

This function is along with the official react-native Image library, and is not be removed.